server/com.dexels.navajo.compiler.tsl/src/com/dexels/navajo/compiler/tsl/internal/BundleQueueComponent.java

```diff
@@ -89,7 +89,7 @@ public synchronized boolean compileScript(final String script, final String path
                 compilationSuccess = false;
                 logger.info("Script compilation failed: {}", script);
             }
-            if (compilationSuccess) {
+            if (compilationSuccess || !skipped.isEmpty()) {
```

Comment:
Based on the code as is, the then-body will **also** execute when **both** `skipped` and `failures` are non-empty.
Is execution in this condition desirable? Or can they never be both non-empty?

Reply:
As far as I know, if skipped is non-empty, compilationSuccess is guaranteed to be false.